content/en/docs/instrumentation/net/exporters.md

```diff
@@ -87,13 +103,7 @@ using var tracerProvider = Sdk.CreateTracerProviderBuilder()
 Use environment variables to set values like headers and an endpoint URL for
 production.
 
-### Using gRPC
-
-You can also use gRPC to send your OTLP data. To do that, use the following:
-
-```csharp
-OtlpExportProtocol.Grpc
-```
+### Note for .NET Core 3.1 and below and gRPC
```

Comment:
This section is valuable, but maybe we should consider removing it.

OpenTelemetry .NET does not formally support .NET 5 and below. It _might_ work, but we don't test on frameworks that have reached end of life. So, keeping this section could cause confusion about what we support.

At a minimum, we should state that we do not officially support .NET Core 3.1.

Reply:
Added it as a note instead, I think it's valueable to be here, and will prompt people to ask questions/dig deeper.